app/assets/javascripts/consumer_role.js

```diff
@@ -11,8 +11,13 @@ $(document).on('change', "#person_no_dc_address, #dependent_no_dc_address, #no_d
 });
 
 $(document).on('ready', function(){
+    if(`EnrollRegistry.feature_enabled?(:ridp_h139) && Rails.env.production? && !(ENV['ENROLL_REVIEW_ENVIRONMENT'] == 'true')`) {
+      var path = "/insured/fdsh_ridp_verifications/new";
```

Comment:
declare the variable outside the `if` block, then reassign within the conditional

``` js
let path;
if (true) {
  path = '';
} else {
  path = '';
}
```